tests/abi/functions.h

```diff
@@ -0,0 +1,84 @@
+#ifndef __REVNG_QA_ABI_FUNCTIONS_H
+#define __REVNG_QA_ABI_FUNCTIONS_H
```

Comment:
I think `__` is a reserved prefix in C. Use `#pragma once`.

Reply:
I REALLY wanted to use `#pragma once` but then I got hesitant about our compiler support. But, ok, since it's no older than `gcc 7.2`, this should be fine, I should have switched right away.

Reply:
[Done](https://github.com/revng/revng-qa/pull/4/commits/cfb308ce4890fb6cc01c353d65ad41be3c0fa26a).